test/e2e/autotls/auto_tls_test.go

```diff
@@ -41,10 +49,14 @@ import (
 	"github.com/ghodss/yaml"
 	cmv1alpha2 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
 	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/kelseyhightower/envconfig"
+
+	"google.golang.org/api/dns/v1"
 )
 
 const (
-	systemNamespace = "knative-serving"
+	systemNamespace   = "knative-serving"
```

Comment:
We shouldn't hardcode this here, I think that `system.Namespace()` has been setup to work.  There are certainly some tests that already have this hardcoded and we should share how they do this, so that it's easier to override.

Reply:
Done